packages/flutter/test/widget/stack_test.dart

```diff
@@ -8,6 +8,15 @@ import 'package:flutter/widgets.dart';
 
 import '../rendering/rendering_tester.dart';
 
+class TestPaintingContext implements PaintingContext {
```

Comment:
we should probably move this to a utility file since we keep introducing it in different places... please file an issue to remind us to do that